src/swagger-generation/output/microsoftgraph-beta-0.1.8-preview.json

```diff
@@ -1429,7 +1439,7 @@
           "groups"
         ],
         "description": "Create or update a group",
-        "operationId": "groups_Put",
+        "operationId": "groups_put",
```

Comment:
Is it really a put or an upsert?

Reply:
The operation http method is `put`. Name doesn't really matter though I can change it to upsert